app/services/project_service.py

```diff
@@ -45,14 +45,34 @@ def fetch(self):
         """
         fetched_projects = self.jira_service.projects()
         persisted_projects = Project.query.all()
+        live_projects = []
+        archived_projects = []
+
+        # TODO: is there a more "Pythonic approach" to this?
```

Comment:
There almost certainly has to be, but I couldn't get things to cooperate since the `Project` object won't always have an `archived` property.